src/js/events/MetronomeActions.js

```diff
@@ -74,10 +65,21 @@ const MetronomeActions = {
               });
               resolve();
             } catch (error) {
-              handleError(error);
+              AppDispatcher.handleServerAction({
+                type: REQUEST_METRONOME_JOBS_ERROR,
+                data: error
+              });
```

Comment:
What about the XHR object in this case? Shall we assume that the consumer of this API should always check if `xhr` is undefined? What about providing an empty object, or an XHR-like object?

What about:

```js
{
  status: 600,
  statusText: 'Exception'
}
```